app/src/router/router.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Src\router;
+
+class Router
+{
+
+    public static function get($route, $callback)
```

Comment:
@jjsimonAB same thing as before retun type, not accessing global variables from within a service